eng/Microsoft.Extensions.targets

```diff
@@ -3,24 +3,47 @@
     <_UseNet6Packages Condition=" '$(TargetFramework.Contains(net6.0))' == 'true' or '$(PackageType)' == 'DotnetPlatform' ">true</_UseNet6Packages>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(_UseNet6Packages)' == 'true' ">
-    <_MicrosoftHostingVersion>6.0.0-preview.5.21301.5</_MicrosoftHostingVersion>
-    <_MicrosoftDependencyInjectionVersion>$(_MicrosoftHostingVersion)</_MicrosoftDependencyInjectionVersion>
-    <_MicrosoftFileProvidersVersion>6.0.0-preview.5.21301.5</_MicrosoftFileProvidersVersion>
-    <_MicrosoftSystemCodeDomVersion>6.0.0-preview.5.21301.5</_MicrosoftSystemCodeDomVersion>
+    <_MicrosoftSystemCodeDomVersion>$(SystemCodeDomPackageVersion)</_MicrosoftSystemCodeDomVersion>
+    <_MicrosoftExtensionsHostingAbstractionsPackageVersion>$(MicrosoftExtensionsHostingAbstractionsPackageVersion)</_MicrosoftExtensionsHostingAbstractionsPackageVersion>
+    <_MicrosoftExtensionsDependencyInjectionAbstractionsPackageVersion>$(MicrosoftExtensionsDependencyInjectionAbstractionsPackageVersion)</_MicrosoftExtensionsDependencyInjectionAbstractionsPackageVersion>
+    <_MicrosoftExtensionsDependencyInjectionPackageVersion>$(MicrosoftExtensionsDependencyInjectionPackageVersion)</_MicrosoftExtensionsDependencyInjectionPackageVersion>
+    <_MicrosoftExtensionsConfigurationAbstractionsPackageVersion>$(MicrosoftExtensionsConfigurationAbstractionsPackageVersion)</_MicrosoftExtensionsConfigurationAbstractionsPackageVersion>
+    <_MicrosoftExtensionsConfigurationPackageVersion>$(MicrosoftExtensionsConfigurationPackageVersion)</_MicrosoftExtensionsConfigurationPackageVersion>
+    <_MicrosoftExtensionsLoggingAbstractionsPackageVersion>$(MicrosoftExtensionsLoggingAbstractionsPackageVersion)</_MicrosoftExtensionsLoggingAbstractionsPackageVersion>
+    <_MicrosoftExtensionsLoggingPackageVersion>$(MicrosoftExtensionsLoggingPackageVersion)</_MicrosoftExtensionsLoggingPackageVersion>
+    <_MicrosoftExtensionsLoggingConsolePackageVersion>$(MicrosoftExtensionsLoggingConsolePackageVersion)</_MicrosoftExtensionsLoggingConsolePackageVersion>
+    <_MicrosoftExtensionsLoggingDebugPackageVersion>$(MicrosoftExtensionsLoggingDebugPackageVersion)</_MicrosoftExtensionsLoggingDebugPackageVersion>
+    <_MicrosoftExtensionsFileProvidersAbstractionsPackageVersion>$(MicrosoftExtensionsFileProvidersAbstractionsPackageVersion)</_MicrosoftExtensionsFileProvidersAbstractionsPackageVersion>
+    <_MicrosoftExtensionsFileProvidersEmbeddedPackageVersion>$(MicrosoftExtensionsFileProvidersEmbeddedPackageVersion)</_MicrosoftExtensionsFileProvidersEmbeddedPackageVersion>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(_UseNet6Packages)' != 'true' ">
     <_MicrosoftHostingVersion>5.0.0</_MicrosoftHostingVersion>
-    <_MicrosoftDependencyInjectionVersion>5.0.1</_MicrosoftDependencyInjectionVersion>
-    <_MicrosoftFileProvidersVersion>5.0.6</_MicrosoftFileProvidersVersion>
     <_MicrosoftSystemCodeDomVersion>4.7.0</_MicrosoftSystemCodeDomVersion>
```

Comment:
Does this one need to be remove and we use `$(SystemCodeDomPackageVersion)` instead?